History.md

```diff
@@ -4,6 +4,7 @@
   * Strip whitespace at end of HTTP headers (#2010)
   * Optimize HTTP parser for JRuby (#2012)
   * Add SSL support for the control app (#2046)
+  * Add requests processed counter to workers (#2057)
```

Comment:
This should be rephrased, as this also works in single mode, which does not use workers.

Maybe "Added a processed_requests counter to Puma.stats"

Reply:
Applied!